.eslintrc.js

```diff
@@ -239,9 +196,6 @@ module.exports = {
     // Disallow tabs.
     "no-tabs": "error",
```

Comment:
Would tabs already be replaced by prettier?

Reply:
There are places like comments where Prettier doesn't touch tabs, while this rule would complain about them.